src/components/field-region/field-region.service.ts

```diff
@@ -91,33 +89,32 @@ export class FieldRegionService {
     });
 
     const result = await this.repo.readOne(id, session);
+    return await this.secure(result, session);
+  }
 
-    if (!result) {
-      throw new NotFoundException(
-        'Could not find field region',
-        'fieldRegion.id'
-      );
-    }
-
-    const secured = await this.authorizationService.secureProperties(
+  private async secure(
+    dto: UnsecuredDto<FieldRegion>,
+    session: Session
+  ): Promise<FieldRegion> {
+    const securedProps = await this.authorizationService.secureProperties(
       FieldRegion,
-      result.propList,
+      dto,
       session
     );
 
     return {
-      ...parseBaseNodeProperties(result.node),
-      ...secured,
+      ...dto,
+      ...securedProps,
       director: {
-        ...secured.director,
-        value: result.directorId,
+        ...securedProps.director,
+        value: dto.director,
       },
       fieldZone: {
-        ...secured.fieldZone,
-        value: result.fieldZoneId,
+        ...securedProps.fieldZone,
+        value: dto.fieldZone,
       },
```

Comment:
These type of explicit call outs are no longer required. Previously there was a mismatch in prop names: the DB returned `fieldZoneId` but our DTO/security code was looking for `fieldZone` (without the `Id` suffix). Now the DB returns with matching keys so these don't need to be "fixed" anymore.

So all of these type of explicit props can be removed. Should just need `dto`, `secured` and `canDelete`